templates/ingress.yaml

```diff
@@ -61,8 +61,8 @@ metadata:
     certmanager.k8s.io/cluster-issuer: {{ $.Values.ingress.tlsClusterIssuer | default "lets-encrypt-http-issuer" | quote }}
     {{- end }}
     {{- if $addIngressClass }}
-    acme.cert-manager.io/http01-ingress-class: {{ $.Values.ingress.ingressClass | default "nginx" | quote }}
-    certmanager.k8s.io/http01-ingress-class: {{ $.Values.ingress.ingressClass | default "nginx" | quote }}
+    acme.cert-manager.io/http01-ingress-class: {{ $.Values.ingress.ingressClassName | default "nginx" | quote }}
+    certmanager.k8s.io/http01-ingress-class: {{ $.Values.ingress.ingressClassName | default "nginx" | quote }}
```

Comment:
Rather than replacing here, could you change it to [coalesce](https://helm.sh/docs/chart_template_guide/function_list/#coalesce) so its backward compatible with anyone still using the `ingressClass` value?